lib/content-services/src/lib/breadcrumb/breadcrumb.component.ts

```diff
@@ -72,6 +82,9 @@ export class BreadcrumbComponent implements OnInit, OnChanges, OnDestroy {
     @Input()
     maxItems: number;
 
+    @Input()
```

Comment:
missing comment which documents that input like for other inputs

Reply:
Yes, especially for ADF as these files are open source so it's helpful for external people to have these documentations.